builder/docker.go

```diff
@@ -59,17 +61,32 @@ func (b *Builder) GetHostAndUpstreamFormats(upstreamVer string) (string, string,
 	return string(hostFormat), upstreamFormat, nil
 }
 
+func getCheckSum(filename string) string {
```

Comment:
I'm not sure how in scope this can be yet, but I think the 'correct' solution here is to get an official signature for the content that we can check with our ClearLinuxRoot.pem, since this isn't part of any manifest. Alternatively, we could check that the mixer running in the container has the same hash as the manifest it shows up in.

Reply:
You'll have to walk me through how that stuff works, but I assume you're talking about verifying the `.sig` files that are published alongside the hashes?

Reply:
Yes, that ^ :)